src/script.ts

```diff
@@ -71,3 +71,26 @@
     return '<%= options.fallback %>'
   }
 })()
+
+// @ts-ignore
+function getStorageValue(storageType, storageKey) {
+  if (typeof window === 'undefined') return null;
```

Comment:
```suggestion
  if (window?.document === 'undefined') return null;
```

The other code will break on build.